submission/races.go

```diff
@@ -55,7 +55,9 @@ type safeSubmissionState struct {
 	mu          sync.Mutex
 	logToGroups map[string]ctpolicy.GroupSet
 	groupNeeds  map[string]int
+	minGroups   int
 
+	groups  map[string]bool
```

Comment:
As there is already a `groupNeeds` map, the naming of `minGroups` and `groups` variables are slightly confusing to me. Adding more comments to the struct may help.

Reply:
Added some comments to make it easier to understand the purpose behind the variables.